packages/wmr/test/fixtures/prerender-resource-fetch/index.js

```diff
@@ -1,4 +1,4 @@
 export async function prerender() {
-	const md = await fetch('content.md').then(res => res.text());
+	const md = await fetch('/content.md').then(res => res.text());
```

Comment:
I forgot to put a leading slash in the last test, so I added it.
This is because when we implemented regular expressions for paths in this PR, relative paths would cause the test to fail, and we believe that relative paths should not be used for access.

Reply:
Now all tests work correctly (currently CI is failing, but this is a problem with wmr's testing infrastructure and will succeed after a few reruns)

Reply:
I don't seem to have the authority to rerun a failed test...